dbaccess/common.go

```diff
@@ -25,10 +23,9 @@ func Save(ctx Context, value interface{}) error {
 		return err
 	}
 
-	dbResult := db.Save(value)
-	dbErrors := dbResult.GetErrors()
-	if httpserverutils.HasDBError(dbErrors) {
-		return httpserverutils.NewErrorFromDBErrors("failed to save object: ", dbErrors)
+	err = db.Insert(value)
```

Comment:
The comment says that `Save` updates, but here you insert. Please update the comment and check that changing it to `Insert`  doesn't harm anything.